### PR TITLE
Refactor all things

### PR DESCRIPTION
**Pull request by AgaK-O**
It was supposed to be a small addition of collection carousel but turned out to be a bigger refactor.

### CODE REVIEW

File: source/includes/_tipser-elements.md [hunk 1]
```diff
@@ -132,16 +133,29 @@ To display `Product` in a compact view, add the `data-tipser-view="compact"` att
...
-Elements with attribute `data-tipser-cid` will be replaced with `Collection` element of given id (value of `data-tipser-cid`). To make the collection items smaller / larger use the `data-tipser-imgsize` attribute with values `0.8` for smaller  and `1.2` for lager product tiles. The default value for imgSize parameter is `1`.
+Elements with attribute `data-tipser-cid` will be replaced with `Collection` element of given id (value of `data-tipser-cid`). 
+
+[![](collection.png)](/images/collcetion.png)
+
+To make the collection items smaller / larger use the `data-tipser-imgsize` attribute with values `0.8` for smaller  and `1.2` for lager product tiles. The default value for imgSize parameter is `1`. When changing the value to `0.8` you get slightly smaller product-cards:
+
+[![](collection-imgSize.png)](/images/collcetion-imgSize.png)
+
+If you'd like the collection of more than several products to take less space, you can display it as one-row only carousel element. To do that, please use `data-tipser-carousel` attribute.
```
Comment (AgaK-O) [inline source/includes/_tipser-elements.md]: This is the initial change for the carousel component.

File: source/includes/_configuration-options.md [hunk 2]
```diff
@@ -120,6 +123,10 @@ Similarly, `hideSimilarProducts` parameter, if set to **true**, can be used to h
 
 [![](widget2.png)](/images/widget2.png)
 
+### Add To Cart Settings
+
+When the`directToChekoutMode` option is set to `true`, after clicking the `buy button` in Product component you go directly to checkout, without using the Cart component.
```
Comment (piotr-sobczyk) [inline source/includes/_configuration-options.md]: without using the Cart component -> without displaying the product dialog

File: source/includes/_overview.md [hunk 3]
```diff
@@ -2,8 +2,8 @@
 
 You can integrate Tipser in your website in multiple ways. The fastest way is to use [Tipser Elements](#tipser-elements) or [Tipser React Elements](#tipser-react-elements) - a complete toolkit of native commerce experience that requires less than 5 minutes of tech setup.
 
-Need more control? Head out to [Tipser SDK](#tipser-sdk) section for low-level JavaScript SDK or go check sections for advanced integrations, such as integrating AMP or setting up tracking with Analytics.
+Need more control? Head out to [Tipser SDK](#tipser-sdk) section for low-level JavaScript SDK or go check sections for advanced integrations, such as integrating [AMP](#amp) or setting up tracking with [Analytics](#analytics).
```
Comment (piotr-sobczyk) [inline source/includes/_overview.md]: ❤️

File: source/includes/_tipser-elements.md [hunk 4]
```diff
@@ -60,7 +60,7 @@ Complete working example could look like:
     <div id="tipser_store"></div>
     <script src="https://cdn.tipser.com/tipser-script/latest.js"></script>
     <script>
-      tipser.elements('5075d7715c3d090a90585e87')
+      tipser.elements('59e86b79b8f3f60a94ecd26a')
```
Comment (piotr-sobczyk) [inline source/includes/_tipser-elements.md]: Why different POS id is used here?

File: source/includes/_tipser-elements.md [hunk 5]
```diff
@@ -105,13 +104,15 @@ Insert below HTML on your page in the place where you want the `Store` element t
 <aside class="notify"><code>Store</code> Element is best inserted as a top level Element on a separate page and should contain the full content area for the best shopping experience.</aside>
 
 ## `Product` Element
-In order to insert a `Product` Element in your content, insert below code in your content.
+In order to insert a `Product` Element in your content, insert this code snippet in your content:
```
Comment (piotr-sobczyk) [inline source/includes/_tipser-elements.md]: this -> the following ?

File: source/includes/_tipser-elements.md [hunk 6]
```diff
@@ -105,13 +104,15 @@ Insert below HTML on your page in the place where you want the `Store` element t
...
 
 ```html
 <div data-tipser-pid="5ba2334a781baa0001ccdf61" />
 ```
 
-Elements with attribute `data-tipser-pid` will be replaced with `Product` Element. Product ID is taken from the value of the attribute. By default a full inline product component is displayed (with product details, unique selling points and variant selection)
+Elements with attribute `data-tipser-pid` will be replaced with `Product` Element. Product ID is taken from the value of the attribute. The code snippet for a certain product can be generated with [tipser tools](#products-collections) automatically.
```
Comment (piotr-sobczyk) [inline source/includes/_tipser-elements.md]: is `#produts-collections` a correct href to tipser-tools section? Shouldn't it be `#tipser-tools`?

File: source/includes/_tipser-elements.md [hunk 7]
```diff
@@ -132,16 +133,29 @@ To display `Product` in a compact view, add the `data-tipser-view="compact"` att
 
 ## `Collection` Element
 
-You can your store in collections. Each collection however can be rendered separately as a get the look widget or a simple product group, depending on your needs. It is all possible through `Collection` Element.
+You can render your collections as `Store`. Each collection, however, can be rendered separately as a simple product group, depending on your needs. It is all possible through the `Collection` Element.
```
Comment (piotr-sobczyk) [inline source/includes/_tipser-elements.md]: Better but still a bit confusing. Do we really need to refer to store component in a section related to collection? Can't we just focus on what a collection is?

File: source/includes/_tipser-elements.md [hunk 8]
```diff
@@ -132,16 +133,29 @@ To display `Product` in a compact view, add the `data-tipser-view="compact"` att
...
+
+[![](collection-imgSize.png)](/images/collcetion-imgSize.png)
+
+If you'd like the collection of more than several products to take less space, you can display it as one-row only carousel element. To do that, please use `data-tipser-carousel` attribute.
 
-> [Open this snippet on Code Pen](https://codepen.io/tipser-tech/pen/YMMKMp)
+[![](collection-carousel.png)](/images/collcetion-carousel.png)
+
+
+> <a href="https://codepen.io/tipser-tech/pen/YMMKMp" target="_blank">Open this snippet on Code Pen</a>
```
Comment (piotr-sobczyk) [inline source/includes/_tipser-elements.md]: I guess you will need to update this codepen to really render the carousel component (currently it doesn't).

File: source/includes/_tipser-elements.md (hunk 4 above)
Comment (AgaK-O) [reply source/includes/_tipser-elements.md]: The new one is Tipser POS Id, the other is of some existing client. I didn't think it very elegant to expose it in our docs even as an example.

**Pull request merged**